addons/web/static/src/js/tools/test_menus.js

```diff
@@ -123,13 +123,15 @@
         console.log("Testing menu", element.innerText.trim(), " ", element.dataset.menuXmlid);
         testedMenus.push(element.dataset.menuXmlid);
         var startActionCount = clientActionCount;
-        element.click();
+        _click($(element));
         var isModal = false;
         return waitForCondition(function() {
             // sometimes, the app is just a modal that needs to be closed
             var $modal = $('.modal[role="dialog"][open="open"]');
             if ($modal.length > 0) {
-                $modal.modal('hide');
+                var $close_button = $('header > button.close');
+                $close_button.focus();
```

Comment:
Is it still needed, despite your `_click` function ?

Reply:
If it is, maybe the `focus` event should be triggered in the `_click` function.

Reply:
Yep, I tested without and it crashes as expected